calyx-py/test/builder_example.py

```diff
@@ -64,7 +63,7 @@ def add_main_component(prog):
     # ANCHOR_END: continuous
 
     # ANCHOR: control
-    main.control += [{update_operands, compute_sum}]
+    main.control = ast.ParComp([update_operands.as_enable(), compute_sum.as_enable()])
```

Comment:
FWIW, there is also `builder.par` now, which is a shorthand for this:
https://github.com/cucapra/calyx/blob/488d1f27fb16d8fdd636f8e9fb46e2d852044661/calyx-py/calyx/builder.py#L669-L671

Reply:
I realized that I erroneously changed the groups to be parallel, not sequential, in [this commit](https://github.com/cucapra/calyx/commit/ced6bf569c5180092b1c96ca093a787ee41876fc) -- totally my bad. I've changed this in #1524.